webpack/assets/stylesheets/application/partials/_password_strength.scss

```diff
@@ -0,0 +1,11 @@
+#password .help-block {
+  .progress .progress-bar .password-verdict {
+    color: #000;
```

Comment:
is there a color-pf for this?

Reply:
Not really, the **blackest** color-pf is:
```scss
$color-pf-black: #030303 !default;
```

For me, it makes sense to use the `$color-pf-black` as our black :+1:
If we decide to take this direction, there are more places we with `#000` that we can migrate.
(prefer to do in other pr)

Any thoughts?

Reply:
I'm ok with a follow up pr - thanks